.Rbuildignore

```diff
@@ -5,3 +5,12 @@
 ^tests/dev
 ^tools
 ^\.covrignore$
+^\.github$
+^docs$
+^scripts$
+^LICENSE\.md$
+^_pkgdown\.yml$
+^rakefile\.rb
+^inst/extdata/RaltegravirSim\.pkml$
+^tests/data/input-data/RaltegravirSim\.pkml$
+^tests/data/input-data/individualPksimSim\.pkml$
```

Comment:
Those pkmls are not used in the unit tests; only in dev-tests.
also not sure why we have RaltegravirSim.pkml twice.

The file under inst/extdata was also much bigger than under tests/data/input-data and I have replaced it with another one.

Later we should keep only one version nayways.